Root.js

```diff
@@ -1,24 +1,25 @@
-import React, { useCallback, useEffect, useState } from "react";
+import React, {useCallback, useEffect, useState} from 'react';
 
-import Storybook from "./storybook";
-import App from "./App";
+// import Storybook from './storybook';
+import App from './App';
 
 const Root = () => {
   const [storybookActive, setStorybookActive] = useState(false);
   const toggleStorybook = useCallback(
     () => setStorybookActive(active => !active),
-    []
+    [],
   );
 
   useEffect(() => {
     if (__DEV__) {
       // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const DevMenu = require("react-native-dev-menu");
-      DevMenu.addItem("Toggle Storybook", toggleStorybook);
+      const DevMenu = require('react-native-dev-menu');
+      DevMenu.addItem('Toggle Storybook', toggleStorybook);
     }
   }, [toggleStorybook]);
 
-  return storybookActive ? <Storybook /> : <App />;
+  // return storybookActive ? <Storybook /> : <App />;
```

Comment:
What does this do? Do you want do disable storybook? Also, don't leave unused code (especially in comments) in production

Reply:
Yes, Anj asked us to comment it for now. We may still need it later.